internal/sql/client.go

```diff
@@ -256,7 +256,7 @@ func (c *client) GetKubernetesProviderAndPermissions(name string) (kubernetes.Pr
 func (c *client) ListKubernetesClustersByApplication(spinnakerApp string) ([]kubernetes.Resource, error) {
 	var rs []kubernetes.Resource
 	db := c.db.Select("account_name, cluster").
-		Where("spinnaker_app = ? AND kind in ('deployment', 'statefulSet', 'replicaSet', 'ingress', 'service', 'daemonSet')",
+		Where("spinnaker_app = ? AND kind in ('Deployment', 'StatefulSet', 'ReplicaSet', 'Ingress', 'Service', 'DaemonSet')",
```

Comment:
I think both in operations should be
```sql
UPPER(kind) in (UPPER('Deployment'), UPPER('StatefulSet'), UPPER('ReplicaSet'), UPPER('Ingress'), UPPER('Service'), UPPER('DaemonSet'))
```
Will need to update the tests for these as well.

Reply:
That would work as well I think.  I'll make the change and add a tests.